examples/cnet-quic/meson.build

```diff
@@ -56,7 +57,7 @@ if quicly_dep.found()
             quic_cflags += arg
         endif
     endforeach
-    quic_cflags += ['-I'+quicly_path+'/include', '-I'+picotls_path+'/include']
+    quic_cflags += ['-I'+quicly_path+'/../include', '-I'+picotls_path+'/../include']
```

Comment:
This change doesn't work if you follow the readme to build the quicly and picotls dirs

Reply:
Needed this to get it build locally, but can remove if it breaks compilation in your case.